alerts/cluster-kube-scheduler-operator/KubeSchedulerDown.md

```diff
@@ -0,0 +1,67 @@
+# KubeSchedulerDown
+
+## Meaning
+
+[This alert][KubeSchedulerDown] is fired when KubeScheduler
```

Comment:
```suggestion
[This alert][KubeSchedulerDown] is fired when the kube-scheduler pods
```

Reply:
We should probably update https://github.com/openshift/cluster-kube-scheduler-operator/blob/master/manifests/0000_90_kube-scheduler-operator_03_servicemonitor.yaml#L49 then.